lms/static/scripts/frontend_apps/components/SubmitGradeForm.js

```diff
@@ -23,6 +23,9 @@ import ValidationMessage from './ValidationMessage';
 // back to [0-1] when saving it to the api service.
 const GRADE_MULTIPLIER = 10;
 
+// Global counter used to create a unique ids
+let idCounter = 0;
```

Comment:
me thinks axe adds multiple instances of the component into each test, so I did this? This seems to be an emerging pattern so maybe there is a way to encapsulate this.

Reply:
Extract a hook perhaps, eg. `const id = useUniqueId(prefix)`?

Reply:
> me thinks axe adds multiple instances of the component into each test, so I did this?

axe tests do run [one at a time](https://github.com/hypothesis/lms/blob/08e1e251fef66d92b72abbc405cc0b5eb183dd8d/lms/static/scripts/test-util/accessibility.js#L61) and elements [are removed](https://github.com/hypothesis/lms/blob/08e1e251fef66d92b72abbc405cc0b5eb183dd8d/lms/static/scripts/test-util/accessibility.js#L38) after each test. However it is possible that axe is remembering IDs it has already seen internally.

In any case, I think it is good practice to design almost all components so that you could have multiple instances in a document, even if the current UI only requires one.